x-pack/plugins/global_search_bar/public/components/search_bar.scss

```diff
@@ -26,23 +26,14 @@
   }
 }
 
-@include euiBreakpoint('m', 'l', 'xl') {
+@include euiBreakpoint('m') {
   .kbnSearchBar {
     width: 400px;
   }
 }
 
 @include euiBreakpoint('l', 'xl') {
-  .kbnSearchBar:focus {
-    animation: kbnAnimateSearchBar $euiAnimSpeedFast forwards;
-  }
-}
-
-@keyframes kbnAnimateSearchBar {
-  from {
-    width: 400px;
-  }
-  to {
+  .kbnSearchBar {
     width: 600px;
```

Comment:
The search bar doesn't need to actually be this large even at large screen sizes. It also then competes with the data search bar.

```suggestion
```

Reply:
The problem is that the results box will still be larger, so if I don't apply the 600px then we have a difference in size between the bar and the results box:

<img width="1045" alt="image" src="https://user-images.githubusercontent.com/25349407/181254142-dc668b02-4ea4-4eb0-a4e3-e7da8caf4740.png">

For this PR I wanted to focus on fixing the **clear button** behaviour. If I have to start tweaking surrounding components (results box, no results, loading spinner) the thing can definitely take longer.

I propose we tackle this on a separate PR / issue if you are okay with that.

Reply:
It's ok that the list is wider than the search bar itself. It contains more information/content than the search input.